web/docs/Backend Starters/Decoupled WordPress/configuring-build-hooks.md

```diff
@@ -19,43 +19,38 @@ frontend; such as content updates to post, or publishing a new page.
 
 ### Creating a Build Hook That Triggers When a Post Is Created Or Updated
 
-1. Navigate to the Pantheon Dashboard
-1. Open the site's settings
-1. Click **Builds** >> **Add Build Hook**
-1. Name your hook and select a branch to target using the **Branch to Build**
-   dropdown. Copy this URL
-1. Navigate to your WordPress development site and open the WP Webhooks plugin
-   dashboard
+1. After generating a build hook in the Pantheon Dashboard, navigate to your
```

Comment:
Be even more ambigous about where the buildhook came from. Write these docs as if Pantheon doesn't exist (weird I know, but I think it helps 😄). Maybe something like the following:

```suggestion
1. After generating a build hook on your build platform,
```

Reply:
Awesome that idea gets the point across. TY